sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/helpers/UniqueIdHelper.java

```diff
@@ -0,0 +1,38 @@
+package com.azure.digitaltwins.core.helpers;
+
+import com.azure.digitaltwins.core.DigitalTwinsClient;
+import com.azure.digitaltwins.core.implementation.models.ErrorResponseException;
+import org.apache.http.HttpStatus;
+
+import java.util.Random;
+import java.util.function.Function;
+
+public class UniqueIdHelper {
```

Comment:
I get the impression that we'll need this logic in the e2e tests and in the samples. If this PR goes in first, I'll try to rewrite my tests to leverage this helper class instead of writing a similar one